docs/Developers/Writing_Good_Commits.rst

```diff
@@ -103,6 +105,9 @@ Example:
     install it by itself.
 
 -  Maximum of 72 chars excluding newline for *each* line.
+   The recommendation is to do this at 72 characters,
+   so that Git has plenty of room to indent text while still
+   keeping everything under 80 characters overall.
```

Comment:
I assume tpope's commit guide refers to http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html